pkg/clusterregistry/options/standalone.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package options contains flags and options for initializing the cluster registry API server.
+package options
+
+import (
+	"github.com/spf13/pflag"
+	"k8s.io/apiserver/pkg/server"
+	"k8s.io/apiserver/pkg/server/options"
+)
+
+// StandaloneServerRunOptions contains runtime options for the cluster registry
+// deployed using the Kubernetes aggregator.
+type StandaloneServerRunOptions struct {
+	*ServerRunOptions
+	authentication *StandaloneAuthenticationOptions
+	authorization  *StandaloneAuthorizationOptions
+}
+
+// NewStandaloneServerRunOptions creates a new StandaloneServerRunOptions
+// object with default values.
+func NewStandaloneServerRunOptions() *StandaloneServerRunOptions {
+	return &StandaloneServerRunOptions{
+		ServerRunOptions: NewServerRunOptions(),
+		authentication:   NewStandaloneAuthenticationOptions().WithAll(),
+		authorization:    NewStandaloneAuthorizationOptions(),
+	}
+}
+
+// AddFlags adds flags for specific StandaloneServerRunOptions fields specified
+// via FlagSet before calling the embedded AddFlags method to add the rest of
```

Comment:
Maybe replace "specified via FlagSet" to "to be added to the FlagSet"?

Reply:
Reworded.